src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -1,48 +1,60 @@
 @import '../theming/theming';
 @import '../style/layout-common';
 
-// Focus indicator styles.
-$mat-focus-indicator-border-radius: 4px;
-$mat-focus-indicator-border-width: 2px;
-$mat-focus-indicator-border-style: solid;
-
 /// Mixin that turns on strong focus indicators.
 ///
 /// @example
 ///   .my-app {
-///     @include mat-strong-focus-indicators();
+///     @include mat-strong-focus-indicators($config);
 ///   }
-@mixin mat-strong-focus-indicators() {
+@mixin mat-strong-focus-indicators($config: ()) {
+  // Default focus indicator config.
+  $default-config: (
+    border-style: solid,
+    border-width: 3px,
+    border-radius: 4px,
+  );
+
+  // Merge default config with user config.
+  $config: map-merge($default-config, $config);
+  $border-style: map-get($config, border-style);
+  $border-width: map-get($config, border-width);
+  $border-radius: map-get($config, border-radius);
 
-  // Base styles for the focus indicators.
+  // Base styles for focus indicators.
   .mat-focus-indicator::before {
     @include mat-fill();
-
-    border-radius: $mat-focus-indicator-border-radius;
-    border: $mat-focus-indicator-border-width $mat-focus-indicator-border-style transparent;
     box-sizing: border-box;
     pointer-events: none;
+    border: $border-width $border-style transparent;
+    border-radius: $border-radius;
   }
 
   // By default, all focus indicators are flush with the bounding box of their
   // host element. For particular elements (listed below), default inset/offset
   // values are necessary to ensure that the focus indicator is sufficiently
   // contrastive and renders appropriately.
 
-  .mat-focus-indicator.mat-button-base::before,
-  .mat-focus-indicator.mat-card::before,
+  .mat-focus-indicator.mat-flat-button::before,
+  .mat-focus-indicator.mat-raised-button::before,
+  .mat-focus-indicator.mat-fab::before,
+  .mat-focus-indicator.mat-mini-fab::before,
   .mat-focus-indicator.mat-chip::before,
   .mat-focus-indicator.mat-sort-header-button::before {
-    margin: $mat-focus-indicator-border-width * -2;
+    margin: -($border-width + 2px);
+  }
+
+  .mat-focus-indicator.mat-stroked-button::before {
+    margin: -($border-width + 3px);
   }
 
   .mat-focus-indicator.mat-calendar-body-cell::before {
-    margin: $mat-focus-indicator-border-width * -1;
+    margin: -$border-width;
   }
 
   .mat-focus-indicator.mat-tab-link::before,
   .mat-focus-indicator.mat-tab-label::before {
-    margin: $mat-focus-indicator-border-width * 2;
+    margin: 5px;
```

Comment:
FYI: The reason why some of these. values are a function of `$border-width` and others are not is because the focus indicator *grows inward* as its width is increased.